glue/viewers/profile/state.py

```diff
@@ -104,18 +113,9 @@ def _reset_x_limits(self, *event):
 
         data = self.reference_data
 
-        if self.x_att in data.pixel_component_ids:
-            x_min, x_max = -0.5, data.shape[self.x_att.axis] - 0.5
-        else:
-            axis = data.world_component_ids.index(self.x_att)
-            axis_view = [0] * data.ndim
-            axis_view[axis] = slice(None)
-            axis_values = data[self.x_att, tuple(axis_view)]
-            x_min, x_max = np.nanmin(axis_values), np.nanmax(axis_values)
-
         with delay_callback(self, 'x_min', 'x_max'):
-            self.x_min = x_min
-            self.x_max = x_max
+            self.x_min = -0.5
+            self.x_max = data.shape[self.x_att_pixel.axis] - 0.5
```

Comment:
Is there a reason for changing the previous behavior here?

Reply:
Yes, it was because I wanted to conform to APE 14's pixel convention: https://docs.astropy.org/en/stable/wcs/wcsapi.html#pixel-conventions-and-definitions